examples/emberjs/.bowerrc

```diff
@@ -0,0 +1,4 @@
+{
+  "directory": "bower_components",
```

Comment:
It can be overwritten in `~/.bowerrc` so Ember CLI outputs this file with the default blueprint so it can guarantee the location.